addon/styles/themes/default/elements/container.overrides

```diff
@@ -958,6 +960,20 @@
 /************************************************************
     Simple layers-style (components/layers-styles/simple)
 ************************************************************/
+.sticky{
+    /* sticky */    
```

Comment:
Unnecessary comment.
Also it will be better to define this `sticky` class somewhere out of `Simple layers-style (components/layers-styles/simple)` block, because `sticky` class is independent from it.